MetaMask/metamask-extension
# Adding Responsive styling to Page component

## **Description**
Adding max width and styling to multichain page for fullscreeen effect, and styling for page component.

## **Manual testing steps**

_1. Run `yarn storybook`_
_2. Go to the story for the multichain Page component, and confirm that the Fullscreen story shows with fullscreen padding_


## **Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change._

### **Before**

_[screenshot]_

### **After**

<img width="985" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/10986371/687ae920-3ef2-4c2e-b698-c086d1567b1a">
## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1551

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
